drivers/ethernet/phy/phy_oa_tc14_plca.c

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2024 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/net/mdio.h>
+#include <zephyr/net/phy.h>
+#include <errno.h>
+
+/* Open Alliance TC14 (10BASE-T1S) PLCA registers */
+#define MDIO_OATC14_PLCA_IDVER  0xca00 /* PLCA ID and version */
+#define MDIO_OATC14_PLCA_CTRL0  0xca01 /* PLCA Control register 0 */
+#define MDIO_OATC14_PLCA_CTRL1  0xca02 /* PLCA Control register 1 */
+#define MDIO_OATC14_PLCA_STATUS 0xca03 /* PLCA Status register */
+#define MDIO_OATC14_PLCA_TOTMR  0xca04 /* PLCA TO Timer register */
+#define MDIO_OATC14_PLCA_BURST  0xca05 /* PLCA BURST mode register */
+
+/* Open Alliance TC14 PLCA IDVER register */
+#define MDIO_OATC14_PLCA_IDM 0xff00 /* PLCA MAP ID */
+#define MDIO_OATC14_PLCA_VER 0x00ff /* PLCA MAP version */
+
```

Comment:
Maybe proper `GENMASK()` would do the trick?

Please use it for all eligible entries.

Reply:
Ah ok, will use GENMASK().